src/services/offer-service.ts

```diff
@@ -24,6 +25,14 @@ export const OfferService = {
   getOffer: async (id: string): Promise<AxiosResponse<Offer>> =>
     await axiosClient.get(createUrlPath(URLs.offers.get, id)),
 
+  getUsersOffers: async (params: GetMyOffersParams): Promise<AxiosResponse> => {
+    const userId = params.id
+    return await axiosClient.get(
+      `${URLs.users.get}/${userId}${URLs.offers.get}`,
```

Comment:
use CreateUrlPath func

Reply:
i cant use this func here. i need to pass more arguments that its expected because of nested routes